src/cli/help.js

```diff
@@ -55,6 +57,7 @@ function usage(print: (string) => void): void {
       graph         build Cred graph from cached plugin data
       site          update your cred site with the latest changes
       credrank      calculate cred scores from existing graph
+      analysis      generates useful data analysis structures
```

Comment:
nit
```suggestion
      analysis      outputs useful cred and grain data for analysis
```

Reply:
`outputs` seems a little misleading as a CLI help tip, since nothing is outputted to the console.

I kept it vague instead of saying cred and grain to leave the door open for any sort of data we want to generate. 

\***slaps roof of analysis command*** "You can fit so many auxilary data structures in this bad boy."